puppet-lint-topscope-variable-check.gemspec

```diff
@@ -17,11 +17,10 @@ Gem::Specification.new do |spec|
     A puppet-lint plugin to check that topscope variable names don't start with ::.
   DESC
 
-  spec.add_dependency             'puppet-lint', '~> 2.0'
-  spec.add_development_dependency 'rake', '~> 12.0'
+  spec.add_dependency             'puppet-lint', '>= 2.0', '< 4'
+  spec.add_development_dependency 'rake', '>= 12.0', '< 14'
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'rspec-collection_matchers', '~> 1.0'
   spec.add_development_dependency 'rspec-its', '~> 1.0'
-  spec.add_development_dependency 'rubocop', '~> 0.58'
```

Comment:
@bastelfreak How come?

Reply:
because we didn't actually run rubocop and the config is totally outdated. and the version 0.58 brings conflicts with newer ruby versions. We're currently discussing how to handle rubocop for puppet-lint + our plugins and want to unify that, after the next puppet-lint release.